messages/sensor/lidar.json

```diff
@@ -0,0 +1,25 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "type": "object",
+    "title": "LIDAR",
+    "description": "Data from a LIDAR sensor",
+    "default": {},
+    "required": [
+        "pose",
+        "points"
+    ],
+    "properties": {
+        "pose": {
+            "type": "object",
+            "title": "Pose",
+            "description": "Pose of the LIDAR sensor",
+            "$ref": "https://raw.githubusercontent.com/brefv/messages/core/pose.json"
+        },
```

Comment:
Is this in global coordinates or in relation to the local vessel coordinate system?